feat: baseline for searcher (#7)
Relewise/relewise-sdk-javascript: README.md

## Usage examples

### Bootstrapping

Whether you need to track or search, you need to start by bootstrapping either the `Tracker` or `Searcher`.

```ts
const tracker = new Tracker(RELEWISE_DATASET_ID, RELEWISE_API_KEY);
const searcher = new Searcher(RELEWISE_DATASET_ID, RELEWISE_API_KEY);
```

Replace the `RELEWISE_DATASET_ID` and `RELEWISE_API_KEY` parameters with your dataset & api key found at [My.Relewise](https://my.relewise.com/developer-settings). 

> After which you have access to various methods depending on the client you bootstrapped.

Reviewer: ```suggestion
After which you have access to various methods depending on the client you've bootstrapped.
```

### Tracking

> If you need to tracking orders, cart changes, or different views on your store, you need to use the `Tracker` 

Reviewer: ```suggestion
If you need to track e.g. Product-, Category or Content Views, track Cart Updates or Completed Orders in your solution, you need to use the `Tracker` 
```

UserFactory.anonymous()
```

### Search

> To sure our fully-fledged search engine, start by bootstraping the `Searcher`-class - see section above on how to bootstrap the `Searcher`.

Reviewer: ```suggestion
To use our fully-fledged search engine, start by bootstraping the `Searcher`-class - see section above on how to bootstrap the `Searcher`.
```
